src/tutorials/narratives-how-to-write.rst

```diff
@@ -0,0 +1,349 @@
+
+  ```auspiceMainDisplayMarkdown
+
+  Markdown content rendered in the right-hand pane of the display, where the dataset would normally be.
+
+  _Note that the dataset URL is still required, although unused._
+
+  ```
+
+Embedding images
+----------------
+
+Publicaly accessible images can be embedded using normal Markdown syntax, for example using `this SEM photo of Yersinia Pestis <https://commons.wikimedia.org/wiki/File:Yersinia_pestis.jpg>`__:
+
+.. code-block:: markdown
+
+  ### Here's a SEM photo of Y. pestis
+
+  ![Y pestis SEM](https://upload.wikimedia.org/wikipedia/commons/thumb/d/d0/Yersinia_pestis.jpg/800px-Yersinia_pestis.jpg)
+
+
+If the image is not publicly accessible via a URL, you can also embed it in the Markdown file itself using base64 encoding [#f3]_ (see `here <https://www.base64-image.de/>`__ for a drag-and-drop utility to convert images to base64):
+
+.. code-block:: markdown
+
+  ![alt text](data:image/png;base64,<image-in-base64-encoding>)
+
+.. _auspice_vs_nextstrain:
+
+Differences between localhost (Auspice) and nextstrain.org
+----------------------------------------------------------
+
+It's possible to run a local instance of Auspice (the phylogenetic visualisation app used in nextstrain.org) and view narratives locally, however there are some differences between this and nextstrain.org which are really easy to get tripped up on!
+Specifically the hostname is *not* used [#f2]_ -- only the pathname is used, and the way the dataset pathname is interpreted is different: Auspice can only access datasets on your computer (in the directory you supply via ``--datasetDir``) whereas nextstrain.org accesses data from :doc:`a range of places </guides/share/index>`.
+
+The following example of a narrative slide may make this difference clearer:
+
+.. code-block:: markdown
+
+  # [Which dataset is this?](https://nextstrain.org/community/inrb-drc/ebola-nord-kivu)
+
+  The dataset displayed here differs if you run it on localhost (auspice)
+  or through nextstrain.org.
+
+  When viewed on nextstrain.org this dataset is sourced from
+  [this GitHub repository](https://github.com/inrb-drc/ebola-nord-kivu)
+  as it is using our
+  [community sharing functionality](https://docs.nextstrain.org/en/latest/guides/share/community-builds.html).
+
+  To work when running on auspice locally, we would need to have a dataset
+  with the following filename: `community_inrb-drc_ebola-nord-kivu.json`.
+
+
+
+This complexity encouraged us to build the interactive debugger we have used in this tutorial and we encourage to use that approach rather than developing your narratives using Auspice locally & then trying to share them through nextstrain.org!
+
+
+Where can I find some example narratives to use as inspiration?
+---------------------------------------------------------------
+
+All of the narratives written by the core Nextstrain team are available in `this GitHub repo <github.com/nextstrain/narratives>`__.
```

Comment:
Ah, I noticed this again during re-review but somehow missed adding the comment for it :facepalm: 

```suggestion
All of the narratives written by the core Nextstrain team are available in `this GitHub repo <https://github.com/nextstrain/narratives>`__.
```

Reply:
Not a problem -- fixed up in https://github.com/nextstrain/docs.nextstrain.org/compare/9425839a96b2cffe9481c7ccf2a4158de08f4909..ea98849a3ee13ec31dfce5a1986449bff43316b7